timvt/endpoints/index.py

```diff
@@ -1,30 +0,0 @@
-"""TiVTiler.index: Index endpoint."""
-
-from typing import List
-
-from ..models.metadata import TableMetadata
-
-from fastapi import APIRouter, Request
-
-router = APIRouter()
-
-
-@router.get(
-    "/index",
-    response_model=List[TableMetadata],
-    tags=["Database"],
-    description="Return available tables.",
-    deprecated=True,
-)
-@router.get(
```

Comment:
Removed this method entirely, it conflicts with the route to request a tilejson for a specific table - `/{table}.json`